AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/sample.json

```diff
@@ -13,11 +13,10 @@
 	{
 		"env": ["source ${ONEAPI_ROOT}/setvars.sh --force",
 			"conda env remove -n user_tensorflow",
-			"conda create -n user_tensorflow -c ${ONEAPI_ROOT}/conda_channel python=`python -V| awk '{print $2}'` -y",
+			"conda create -n user-tensorflow --clone tensorflow -y",
```

Comment:
@xiguiw Is it ``user-tensorflow`` or ``user_tensorflow``

Reply:
@jimmytwei  

Sorry for the typo.  
It is user_tensorflow. Corrected.